ui/temboardui/model/orm.py

```diff
@@ -234,6 +234,25 @@ class Roles(Model):
     def count(cls):
         return text(QUERIES["users-count"]).columns(count=types.Integer)
 
+    @classmethod
+    def all(cls):
+        return (
+            Query(cls)
+            .from_statement(
+                text(QUERIES["roles-all"]).columns(
```

Comment:
I wonder if this is mandatory to provide a list of columns here since the query itself includes the same list.

Reply:
Oui, on doit déclarer les colonnes retournées par une requête texte.

Reply:
Désolé d'insister mais j'ai essayer sans et je ne vois aucune différence.

Reply:
Corrigé, merci d'avoir testé !